lib/rubocop/cop/lint/ruby2_keywords.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop looks for `ruby2_keywords` calls for methods that do not need it.
+      #
+      # `ruby2_keywords` should only be called on methods that accept an argument splat
+      # (`*args`) but do not explicit keyword arguments (`k:` or `k: true`) or
+      # a keyword splat (`**kwargs`).
+      #
+      # @example `ruby2_keywords` given a `def`
```

Comment:
`@example` directive is used to represent `EnforcedStyle` (and similar style option). Can you use source comment for explanation to examples? :-)